src/mame/drivers/namcos2.cpp

```diff
@@ -20,12 +20,15 @@ Final Lap Notes:
     To move through self test options, press gas pedal and change gear shift from low to high
     To change an option, move gear shift from low to high without touching the gas pedal
 
-known issues:
+TODO:
+	- Verify below still occur
+
+	General
     - sprite/tilemap orthogonality needed
-    - bad road colors in Final Lap and Suzuka series
+    - bad road colors in Final Lap and Suzuka series (just an overall gamma issue?)
```

Comment:
Isn't this just leftover from earlier days where road were rainbow colored?
![suzuk8h2](https://user-images.githubusercontent.com/9251271/125648142-4aa36288-9d9e-48e8-94b1-dc3240f2f496.gif)

Reply:
probably, hence adding the note at the top to verify them.